### PR TITLE
Add org.gtk.Gtk3theme.Yaru-remix-light

### PR DESCRIPTION
**Pull request by Muqtxdir**
hi,
This is the flatpak version of [Yaru-remix](https://github.com/Muqtxdir/yaru-remix) theme light variant.

### CODE REVIEW

**Comment by Muqtxdir** [comment]
bot, build org.gtk.Gtk3theme.Yaru-remix-light

File: org.gtk.Gtk3theme.Yaru-remix-light.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+   "id":"org.gtk.Gtk3theme.Yaru-remix-light",
+   "branch":"3.22",
+   "runtime":"org.freedesktop.Sdk",
+   "build-extension":true,
+   "sdk":"org.freedesktop.Sdk",
+   "runtime-version":"19.08",
```
Comment (barthalion) [inline org.gtk.Gtk3theme.Yaru-remix-light.json:7]:
```suggestion
   "runtime-version":"20.08",
```

Comment (barthalion) [reply org.gtk.Gtk3theme.Yaru-remix-light.json:7]: Also applies to other variants.

Comment (Muqtxdir) [reply org.gtk.Gtk3theme.Yaru-remix-light.json]: ok, changed it

**Comment by Muqtxdir** [comment]
Queued test build for org.gtk.Gtk3theme.Yaru-remix-light.

**Comment by Muqtxdir** [comment]
bot, build org.gtk.Gtk3theme.Yaru-remix-light

**Comment by barthalion** [comment]
Yeah, it's fine. The new repositories is where you should push any changes.

**Pull request closed without merging**